extensions/ql-vscode/package.json

```
@@ -14,7 +14,7 @@
   },
   "engines": {
     "vscode": "^1.59.0",
-    "node": ">=16.11.25",
+    "node": "^16.11.25",
```

This should match the node version in the nvmrc file.
```suggestion
    "node": "^16.13.0",
```